notifications/2024-10-28_CHG0064176.md

```diff
@@ -0,0 +1,49 @@
+@all
+**What is happening?**
+
+CHG0064176 - CITZ - MCS - Drain and reboot SILVER and GOLD DELL nodes to apply a firmware update
+
+**When?**
+
+Starting at 9AM Tuesday October 29th and completing before 4:30 PM on Thursday October 31st.
+
+**Will there be an impact on the Platform apps?**
+
+Pods will be drained from the affected nodes, similar to OpenShift patching. The nodes will then have their firmware updated to a newer more stable version and returned to service. The SILVER nodes will be completed first and tested before moving onto the GOLD cluster.
+
+Nodes:
+
+- mcs-gold-app-01.dmz
+- mcs-gold-app-02.dmz
+- mcs-gold-app-03.dmz
+- mcs-gold-app-04.dmz
+- mcs-gold-app-05.dmz
+- mcs-gold-app-06.dmz
+- mcs-gold-app-07.dmz
+- mcs-gold-infra-01.dmz
+- mcs-gold-infra-02.dmz
+- mcs-gold-infra-03.dmz
+- mcs-gold-master-01.dmz
+- mcs-gold-master-02.dmz
+- mcs-gold-master-03.dmz
+- mcs-silver-app-36.dmz
```

Comment:
Suggest re-arranging the list to put the Silver nodes first

Reply:
Reordered.